client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -84,7 +85,7 @@ class TaxonomyManagerListItem extends Component {
 					<Gridicon
```

Comment:
@timmyc Is this any particular reason ? I'll try the EllipsisMenu to see if it fits